i18n/en/docusaurus-plugin-content-docs/current/guides/tech/with-nuxtjs.mdx

```diff
@@ -0,0 +1,121 @@
+---
+sidebar_position: 10
+---
+# Use with NuxtJS
+
+It is possible to implement FSD in a NuxtJS project, but conflicts arise due to differences between the NuxtJS project structure requirements and FSD principles: 
+
+- Initially, NuxtJS offers a project file structure without a `src` folder, i.e. in the root of the project.
+- The file routing is in the `pages` folder, while in FSD this folder is reserved for the flat slice structure.
+
+
+## Adding an alias for the `rc` directory
+
+Add an `alias` object to your config:
+
+```ts
+export default defineNuxtConfig({
+  devtools: { enabled: true },      // Not related to FSD, enabled at project startup
+  alias: {
+    "@": '~/src'
+  },
+})
+```
+
+## Move file routing to `src/app`.
+
+First of all, create a `src` directory in the root of your project, and create app and pages layers inside this directory.
+Thus, your file structure should look like this:
+
+```sh
+├─── src
+│ ├─── app           
+│ ├─── pages                        # The pages folder assigned to FSD
+```
+
+In order for NuxtJS to use the `app` layer for file routing, you need to modify `nuxt.config.ts` as follows:
```

Comment:
issue: this kind of structure prevents you from using the App layer for other purposes, or forces the use of underscores

suggestion: maybe we can put routes in `app/routes`? That way we can still make folders like `app/layouts`, `app/styles`, etc.